src/Application.php

```diff
@@ -26,4 +26,19 @@ public function run(?array $argv = null): void
         $this->environment->build();
         parent::run($argv);
     }
+
+    public static function getVersion(): string
+    {
+        $version = trim(@file_get_contents(__DIR__ . '/../home/build'));
```

Comment:
could probably pull this from the env in the build

Reply:
`composer compile <version>` embeds the version in that file in the phar, and `<version>` comes from GH action environment

Reply:
you can replace this with:

https://github.com/phpbench/phpbench/blob/414f462c2ee40a6b30ffafbe0c4ee760e75780f8/lib/PhpBench.php#L40

I think

Reply:
as discussed directly, I am not sure either way is particularly better or worse, I do feel our current implementation is easy to follow without having to be aware of box's placeholder replacement functionality so I will leave as is for now